collected/icla/fblaise/committer.properties

```diff
@@ -0,0 +1,3 @@
+name=Fred Blaise
+email=fred.blaise@gmail.com
```

Comment:
In the past I have recommended that those signing a Company CLA use their work email address in this field.

Reply:
I thought about that. Sorry, I probably missed it somewhere when reading up instructions.
Forced pushed over the change in my branch. Thanks.